contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.5.0--3.6.0.sql

```diff
@@ -62,3 +62,64 @@ CALL sys.analyze_babelfish_catalogs();
 
 -- Reset search_path to not affect any subsequent scripts
 SELECT set_config('search_path', trim(leading 'sys, ' from current_setting('search_path')), false);
+
+CREATE OR REPLACE FUNCTION sys.sp_tables_internal(
```

Comment:
Please move the SQL upgrade changes before line no. 56, SQL upgrade changes should be before the drop of temporary procedure and reset of search path. Can you also please raise PR for same in 4x?

Reply:
Updated the patch and filed #2482 for 4x.